docs/source/scripts_and_their_functions.rst

```diff
@@ -0,0 +1,27 @@
+Scripts and their functions
```

Comment:
Nice to move this here. I guess there'll be an EBU-TT-D encoder one too at some point?